### ISSUE TITLE
Info manuals not visible

### ISSUE THREAD

**Issue by Triagle** [fejfighter/pgtk-emacs-flatpak#11]
Firstly want to say thank you for building this, my experience with this flatpak has been really solid. When I invoke the info manual with `M-x info` I am missing the emacs manual that would usually be present. This problem persists even with my config disabled via passing the `-Q` flag. The emacs info manuals are installed to the `/app/share/info` directory which is populated in the `Info-directory-list` after invoking `(info-initialize)`. If I invoke info on the emacs manual in this directory manually (using eshell: `info emacs.info.gz`) it does work, but it does not then populate the directory node of `M-x info`.

**Comment by maddovr** [comment]
It's still not fixed actually, emacs info manual is still not there, just installed packages manual.

**Comment by Triagle** [comment]
Yup, I tried the fix yesterday and it did not work for me. I thought it was because the "env" field was placed in the wrong place, because I kept getting warnings about it. I tried moving it to the build-options section; it did fix the warning but it did not fix the build. 

Somehow, the build is installing the info files to the correct location, but not calling `install-info` to populate the directory node during the build. My best guess is maybe install-info is not available in the flatpak build sandbox. Admittedly I don't really know anything about how flatpak builds packages, so perhaps it doesn't sandbox the building process at all.

**Comment by fejfighter** [comment]
@Triagle @maddovr What distro are you running?

I generally use this on a normal Fedora workstation but I have seen a few things that are different on silverblue.

it seems that there are a few "leaky" bits on workstation that don'` occur on silverblue so I get more access to host files than normal

Longer term I do want to move back to silverblue when there's a sway spin but I have not had to time to put one together.

**Comment by Triagle** [comment]
I'm running this in fedora silverblue, and it's built in a toolbox container. The container does have `install-info`. I tried running the build step without installing, then mucking around with the files to build the directory node manually in the build-dir, and finally installing the flatpak as normal. Flatpak just pulls out a cached copy of the build before installing, so my manual changes don't reflect in the final flatpak. Not sure where it's getting it from.

**Comment by fejfighter** [comment]
I have been able to replicated this on my silverblue system

I can see the same behavior
it's not really ignoring the info bundles on the path (as straight extends that variable) it's seemingly ignoring any gzipped info bundles, will need to look closer tomorrow
